testing/testcases.py

```diff
@@ -0,0 +1,37 @@
+# =================================================================================================
+#                                  All Rights Reserved.
+# =================================================================================================
+# File description:
+#           Make all unit tests are inherited from this class
+#           so that all unit tests can share common methods
+#
+# =================================================================================================
+#    Date      Name                    Description of Change
+# 06-Sep-2021  Wayne Shih              Initial create
+# $HISTORY$
+# =================================================================================================
+
+
+from django.test import TestCase as DjangoTestCase
+from django.contrib.auth.models import User
+from tweets.models import Tweet
+
+
+class TestCase(DjangoTestCase):
+
+    def create_user(self, username, email=None, password=None):
+        if password is None:
+            password = 'default_generic_password'
+        if email is None:
+            email = f'{username}@django-twitter.com'
+        return User.objects.create_user(username, email, password)
+
```

Comment:
Since the intention here is to make all unit tests are inherited from this class so that all unit tests can share common methods, it might need to use this create_user() for accounts api unit tests as well.

Reply:
Resolved as suggestion!